core/src/main/scala/org/bitcoins/core/wallet/utxo/AddressTag.scala

```diff
@@ -0,0 +1,14 @@
+package org.bitcoins.core.wallet.utxo
+
+case class AddressTagType(name: String)
```

Comment:
I don't think this type is much better than a `String` and should probably be an `ADT ` instead

Reply:
https://github.com/bitcoin-s/bitcoin-s/pull/1320#discussion_r406342504